zingolib/src/commands/utils.rs

```diff
@@ -131,13 +140,16 @@ pub(super) fn parse_send_all_args(
 pub(super) fn parse_spendable_balance_args(
     args: &[&str],
     chain: &ChainType,
-) -> Result<Address, CommandError> {
+) -> Result<(Address, bool), CommandError> {
     let address: Address;
 
-    if args.len() != 1 {
+    let mut zennies_for_zingo = false;
+    if args.len() > 2 {
         return Err(CommandError::InvalidArguments);
     }
-
+    if args.len() == 2 && args[1] == "true" {
+        zennies_for_zingo = true;
+    }
```

Comment:
Rather than set false initially and set true if true is correctly spelt etc. I think it safer to move this logic into the next block where we check if the first argument is a zcash address (not a json string) and parse true OR false there OR error

Reply:
This also requires us to move the zennies for zingo into json which I think is correct for the json option

Reply:
I believe this is now ready for review.